tests/suites/test_suite_platform.function

```diff
@@ -32,91 +32,106 @@ void sleep_ms(int milliseconds)
     usleep(milliseconds * 1000);
 #endif
 }
-#endif
 
+#endif
 /* END_HEADER */
 
 /* BEGIN_DEPENDENCIES */
 
 /* END_DEPENDENCIES */
 
 /* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
-void time_get_milliseconds()
-{
-    mbedtls_ms_time_t current = mbedtls_ms_time();
-    (void) current;
-    /* This goto is added to avoid warnings from the generated code. */
-    goto exit;
-}
-/* END_CASE */
-
-/* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
-void time_get_seconds()
-{
-    mbedtls_time_t current = mbedtls_time(NULL);
-    (void) current;
-    /* This goto is added to avoid warnings from the generated code. */
-    goto exit;
-}
-/* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
+/* `time_delay_milliseconds()` and `time_delay_seconds()` might be fail due to
+ * `signal interrupt`, `discontinues jumps` and `incremental adjustment`. The
+ * test functions run 3 iteration to ignore the failures caused by those
+ * conditions.
```

Comment:
I'm trying to understand the assumptions here. Are we assuming that only one of the three problems will occur? Or that each problem will occur at most once? Or a different assumption?

Reply:
Each problem will occur at most once